discovery_engine_core/providers/src/lib.rs

```diff
@@ -101,6 +104,15 @@ pub trait TrendingTopicsProvider: Send + Sync {
     ) -> Result<Vec<TrendingTopic>, Error>;
 }
 
+/// Abstraction over a provider for similar news.
```

Comment:
:art: the naming is somewhat redundant, because this is a trait after all. i saw that the doc naming is also used for the other traits, maybe it's worth updating their docs as well.
```suggestion
/// A provider for similar news.
```

Reply:
thanks, I cut out the unnecessary words from all the provider docstrings.